bp_params->target_pixel_clock_100hz changed to bp_params->target_pixel_clock
----
--- a/drivers/gpu/drm/amd/display/dc/bios/command_table2.c.orig	2019-03-03 15:21:29.000000000 -0800
+++ b/drivers/gpu/drm/amd/display/dc/bios/command_table2.c	2019-05-12 01:03:57.809033141 -0700
@@ -309,9 +309,10 @@ static enum bp_result set_pixel_clock_v7
 			(uint8_t) bp->cmd_helper->
 				transmitter_color_depth_to_atom(
 					bp_params->color_depth);
-		DC_LOG_BIOS("%s:program display clock = %d"\
-				"colorDepth = %d\n", __func__,\
-				bp_params->target_pixel_clock, bp_params->color_depth);
+		DC_LOG_BIOS("%s:program display clock = %d, tg = %d, pll = %d, "\
+				"colorDepth = %d\n", __func__,
+				bp_params->target_pixel_clock, (int)controller_id,
+				pll_id, bp_params->color_depth);
 
 		if (bp_params->flags.FORCE_PROGRAMMING_OF_PLL)
 			clk.miscinfo |= PIXEL_CLOCK_V7_MISC_FORCE_PROG_PPLL;
